### ISSUE TITLE
Feature Request: Multiple Devices

### ISSUE THREAD

**Issue by toastwaffle** [aluzzardi/pam_usb#8]
Would it be possible to have more than one device for one user? I'd like to use this for two factor auth, but I'd like to be able to have a spare key in case I lose my original one.

**Comment by dj95** [comment]
If this issue is still recent and you're looking forward to multiple devices, you can check out my fork of pamusb in a few weeks. I want to implement this feature into the pamusb-agent, but first of all it should be cleaned and ported to Python 3. The portation to Python 3 is mostly done, so refactoring and code cleaning can start.

**Comment by Sjd-Risca** [comment]
Yes, for me it is still a big issue. Please let me know when you expect it would be ready for testing and debugging!

Thanks a lot.

**Comment by dj95** [comment]
I implemented this feature, but it needs a little bit of clean up. There's also some bugfixing from the original pamusb-agent needed.

Feel free to test it and send me feedback and bug reports!

You're able to find the version [here](https://github.com/dj95/pam_usb/tree/pamusb-multiple-devices).
